### PR TITLE
Add create options

### PR DESCRIPTION
**Pull request by samuel-w**
I have an issue before I continue on this:
Should I create a new tab for just the options, as the putting it in the schedule tab as suggested in #274 makes the schedule tab cluttered and create options aren't really part of scheduling, but I also don't want to end up with an interface with 10 tabs

Now that I think about it, I could rename the new tab to Backup Options, and move the shell commands there as well

Looking for input, thanks

### CODE REVIEW

**Comment by samuel-w** [comment]
initialui
This is my current UI setup. Should I break it into tabs like it appears in the [documentation](https://borgbackup.readthedocs.io/en/stable/usage/create.html)?

**Comment by m3nu** [comment]
It could be good to have those options somewhere hidden under advanced, but Vorta's goal is not to replicate every Borg option 1:1.

Some of the options in your screenshot are also related to the source tab/exclusions and should go there if they are added.

I also don't think that we should add a new tab to mirror a specific Borg subcommand. A GUI will present information differently than a CLI tool. By that logic we should also move the *Sources* tab to *Create*, which would worsen the UI.

So for each option you should consider, *if* it's needed in the GUI and *where* it should go. E.g. we had very few requests to support storing metadata change times or whatever. 😁

In addition the way you set up the changes, no options are ever saved. So they would only persist until the window is closed. That may be confusing.

To summarize, yes we can add new options if they add value to a significant group of users and are integrated with the UI structure. But we won't implement every single option the Borg CLI provides.

**Comment by m3nu** [comment]
Let me also link to this piece, since it kinda fits here:

**Comment by m3nu** [comment]
Related: https://blog.codinghorror.com/this-is-what-happens-when-you-let-developers-create-ui/

😜

**Comment by samuel-w** [comment]
I didn't implement saving just in case I had to change things. Going to reconsider my approach. Thanks.

**Comment by samuel-w** [comment]
I'll probably just stick a button under the excludes in the sources tab
